test/utils/CommonActions.js

```diff
@@ -127,16 +118,6 @@ class CommonActions {
         }
     }
 
-    /**
-     * @param message popup message.
-     * @param closeButton close button on popup message at Classic Theme.
-     */
-    static closePopMessage(message, closeButton) {
```

Comment:
Are you sure? this method is to close Spam message after choose any section on classic theme.

Reply:
sure it is not neccesary.I tried it